lua/gitlab/server.lua

```diff
@@ -84,9 +84,14 @@ end
 M.build = function(override)
   local file_path = u.current_file_path()
   local parent_dir = vim.fn.fnamemodify(file_path, ":h:h:h:h")
+  state.settings.root_path = parent_dir
+
+  if state.settings.bin ~= nil then
+    u.notify(string.format("skipping server intallation, using: %s", state.settings.bin), vim.log.levels.INFO)
```

Comment:
A typo, and messages from the plugin typically start with capital letters:
```suggestion
    u.notify(string.format("Skipping server installation, using: %s", state.settings.bin), vim.log.levels.INFO)
```